### DISCUSSION TITLE
New "expectedPublicationDate" field in the "search-esentool" operation

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#842] [Q&A] [answered]
After installing a new release in eNotices2 Preview environment we have seen that the "search-esentool" operation returns a new field "expectedPublicationDate". This field causes problems in our system. Have you been notified about this new field? Is there any intention to add it to the production environment?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
For that not SDK related question, please get in touch with the helpdesk: ted@publications.europa.eu

**Reply by AlexAS81** [reply]
Does a change in the web services have anything to do with the SDK?

**Reply by YvesJo** [reply]
it does not, so please get in touch with the helpdesk
